addons/account/models/account_payment_term.py

```diff
@@ -15,15 +15,21 @@ class AccountPaymentTerm(models.Model):
     def _default_line_ids(self):
         return [Command.create({'value': 'balance', 'value_amount': 0.0, 'days': 0, 'end_month': False})]
 
+    def _default_example_amount(self):
+        return self._context.get('default_amount_total') or 100  # Force default value if the context is not set or set to False
```

Comment:
If the context was not set, we could use `default=100` on the field. It is really for when the context is voiding the value explicitly that we need this function